release-notes/aws/10.0.0-rc1

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 10.0.0-rc1 for AWS is now active for you! :zap:
+
+We are pleased to announce that Node Pools is now released! 
+
+Please see here the supported material which we recomend reading before getting started: https://docs.giantswarm.io/basics/nodepools
```

Comment:
```suggestion
Please see here the supported material which we recommend reading before getting started: https://docs.giantswarm.io/basics/nodepools
```

Reply:
The link is currently broken. I've asked for help here: https://gigantic.slack.com/archives/C02NGJTJN/p1576667008001000